Lenovo/Z5s/res/values/notch.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/*
+** Copyright 2017, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="rounded_corner_radius">90px</dimen>
+    <dimen name="status_bar_height">87px</dimen>
+    <string translatable="false" name="config_mainBuiltInDisplayCutout">M -80,0 L -50,90 L 50,90 L 80,0 Z</string>
```

Comment:
I don't actually know how to interpret that path, but I find it curious that it only has coordinates divisible by 10, but the `status_bar_height` is not divisible by 10. Shouldn't the cutout height be equal to `status_bar_height`?